src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -22,13 +23,20 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+        // Reads API Key and HTTP referer from file
+        ClassLoader classLoader = MapServlet.class.getClassLoader();
+        File apiKeyFile = new File(classLoader.getResource("keys.txt").getFile());
+        Scanner scanner = new Scanner(apiKeyFile);
+        final String apiKey = scanner.nextLine();
+        final String referer = scanner.nextLine();
```

Comment:
you can get rid of this line. I used it only for the API call, but I don't think you'll it need it for maps